templates/user/settings/keys_gpg.tmpl

```diff
@@ -18,11 +18,11 @@
 					<p>{{.locale.Tr "settings.gpg_token_required"}}</p>
 				</div>
 				<div class="field">
-					<label for="token">{{.locale.Tr "setting.gpg_token"}}
+					<label for="token">{{.locale.Tr "settings.gpg_token"}}
 					<input readonly="" value="{{.TokenToSign}}">
 					<div class="help">
 						<p>{{.locale.Tr "settings.gpg_token_help"}}</p>
-						<p><code>{{$.locale.Tr "settings.gpg_token_code" .TokenToSign .PaddedKeyID}}</code></p>
+						<p><code>{{$.locale.Tr "settings.gpg_token_code" .TokenToSign .KeyID}}</code></p>
```

Comment:
Wasn't that explicitly changed to only display the first 16 chars?
But apparently, this edge case was overlooked, so I'm fine with the proposed solution.

Reply:
Looking at #20878, yeah that's indeed the case... If we want to have the same behavior, we would need to change https://github.com/go-gitea/gitea/blob/e81ccc406bf723a5a58d685e7782f281736affd4/models/asymkey/gpg_key_add.go#L96 to return the padded key ID. Or move the padding code to a util.

CC @zeripath what's your thoughts?

Reply:
You're still gonna need the padding as keys that start with an initial 0 exist